From 5ae94d1804a2c44245c3c026e430ff12b6e4a113 Mon Sep 17 00:00:00 2001
From: BlackEagle <ike.devolder@gmail.com>
Date: Thu, 11 Dec 2014 13:47:28 +0100
Subject: [PATCH] atlas post-processor unknown atlas.UploadartifactOpts field
 BuildId

in https://github.com/hashicorp/atlas-go/blob/master/v1/artifact.go
we can see it is BuildID and not BuildId

Signed-off-by: BlackEagle <ike.devolder@gmail.com>
---
 post-processor/atlas/post-processor.go | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/post-processor/atlas/post-processor.go b/post-processor/atlas/post-processor.go
index 530c4e5..f66eab9 100644
--- a/post-processor/atlas/post-processor.go
+++ b/post-processor/atlas/post-processor.go
@@ -157,7 +157,7 @@ func (p *PostProcessor) PostProcess(ui packer.Ui, artifact packer.Artifact) (pac
 		Type:     p.config.Type,
 		ID:       artifact.Id(),
 		Metadata: p.metadata(artifact),
-		BuildId:  p.config.buildId,
+		BuildID:  p.config.buildId,
 	}
 
 	if fs := artifact.Files(); len(fs) > 0 {
--
2.1.3
